core/arch/arm/include/sm/optee_smc.h

```diff
@@ -559,6 +559,32 @@
 /* See OPTEE_SMC_CALL_WITH_REGD_ARG above */
 #define OPTEE_SMC_FUNCID_CALL_WITH_REGD_ARG	U(19)
 
+/*
+ * Access to the secure watchdog.
+ *
+ * Call requests usage:
+ * a0	SMC Function ID, OPTEE_SMC_FUNCID_WATCHDOG
+ * a1	Watchdog command
+ *	0 : Init watchdog
+ *	1 : Set timeout
+ *	2 : Enable watchdog
+ *	3 : Ping the watchdog for refresh
+ *	4 : Get time left
+ * a2-6	Not used
```

Comment:
How is the timeout passed (command 1) and how is the time left returned (command 4)?

Reply:
Sorry, missed some part, I'll update it quickly